r_code/jackknife_te.R

```diff
@@ -1,5 +1,12 @@
 pacman::p_load(dplyr, furrr, stringr, tidyr, tibble, resample)
 
+.CarefulDivision <- function(num, denom){
+  if(is.na(denom) | denom==0){
+    return(NA)
+  } else{
+    return(num/denom)
+  }
+}
```

Comment:
```suggestion
# Return the division or, to avoid division by zero, NA.
.CarefulDivision <- function(num, denom) {
  if (is.na(denom) | denom == 0) {
    return(NA)
  } else {
    return (num/denom)
  }
}
```